CMSgov/easi-app
# [EASI-2113] Verify User - GRT Admin

# EASI-2113

## Changes and Description

- Warning notification for associated intake emails that are not linked as EUA contacts within EASI
- Ability to do a CEDAR lookup on the unverified emails to accurately associated the email with a contact
- Save the verified contact as a part of the intake contact list
- Unit tests



## How to test this change

- [Review "request business case" action screen](http://localhost:3000/governance-review-team/e7f1c09e-f8e0-4fe6-b9b2-f88ad410ce83/actions/need-biz-case)
- Recipients should show as unverified
- Verify recipients using CEDAR contact select field
- Ensure emails send to verified recipients

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
